style.css

```diff
@@ -0,0 +1,59 @@
+
+.business-services{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 100px 0px;
+}
+.header-font{
+    color: #393939;
+    font-size: 36px;
+    font-family: 'Raleway', sans-serif;
+    font-weight: bolder;
+
+}
+.font-design
+    {
+    color: #626262;
+    font-size: 16px;
+    font-family: 'Raleway', sans-serif;
+    line-height: 25px;
+    text-align: center;
+}
+.description{
+    display: block;
+    text-align: center;
+    width: 530px;
```

Comment:
here, when I use mobile view, the description is not responsive. because you gave it a fixed width.
you can solve this by creating a media query and change the width of it.
or you can remove the fixed width, and give it some padding from the right and left.